django/pyproject.toml

```diff
@@ -25,13 +25,13 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "~3.10.0"
+python = "~3.11.0"
 django = ">=4.1,<4.2"
 psycopg2 = "~2.9.3"
 redis = { version = "~4.3.4", extras = [ "hiredis" ] }
 uritemplate = "^4.1.1"
 iso3166 = "^2.1.1"
-rio-tiler = "^3.1.6"
+rio-tiler = "5.0.3"
```

Comment:
Note, this one package had t be updated to be installable on Python 3.11. We should do an upgrade of the rest our python dependencies at some point.